docs/monoDrive_home/Camera.md

```diff
@@ -161,27 +227,35 @@ Provides a grayscale camera stream.
        "x": 512.0,
        "y": 512.0
      },
-    "max_distance": 50000.0,
-    "dynamic_range":  50,
     "fov": 60.0,
-    "focal_length": 9.0,
-    "fstop": 1.4,
-    "min_shutter":  0.000500,
-    "max_shutter":  0.001400,
-    "sensor_size":  9.07,
-   "channels" : "rgba"
+    "debug_draw": false,
+    "annotation": false,
+    "include_tags": false,
+    "include_obb": false,
+    "cull_partial_frame": false,
+    "far_plane": 5000.0,
+    "desired_tags": [
+      "dynamic"
+    ]
  }
-
 ]
 ```
 
-<p>&nbsp;</p>
+These configuration values are the as the RGB camera. Note that the output format
```

Comment:
These configuration values are the same as the RGB camera.

Reply:
Fixed.